hickle/helpers.py

```diff
@@ -89,18 +95,22 @@ def check_iterable_item_type(iter_obj):
 
     Returns:
         iter_type: type of item contained within the iterable. If
-                   the iterable has many types, a boolean False is returned instead.
+            the iterable has many types, a boolean False is returned instead.
 
     References:
-    http://stackoverflow.com/questions/13252333/python-check-if-all-elements-of-a-list-are-the-same-type
```

Comment:
Not going to rally against this, but keeping a URL on one line seems far preferable to adding in a break to appease a dogmatic linter. Linux Kernel is heading in this direction, prefer the 80-character limit but use common sense -- https://www.kernel.org/doc/html/v5.6/process/coding-style.html#breaking-long-lines-and-strings